app/components/avo/views/resource_edit_component.rb

```diff
@@ -3,13 +3,16 @@
 class Avo::Views::ResourceEditComponent < Avo::ResourceComponent
   include Avo::ApplicationHelper
 
-  attr_reader :actions, :display_breadcrumbs
+  prop :resource, _Nilable(Avo::BaseResource)
+  prop :record, _Nilable(ActiveRecord::Base)
+  prop :actions, _Array(Avo::BaseAction), default: [].freeze
+  prop :view, Avo::ViewInquirer do |value|
+    value = :edit if value.nil?
```

Comment:
Can we use `default` here?
```suggestion
  prop :view, Avo::ViewInquirer, default: :edit do |value|
```